libsolidity/interface/CompilerStack.cpp

```diff
@@ -285,17 +285,16 @@ bool CompilerStack::analyze()
 		// the special variables "this" and "super" must be set appropriately.
 		for (Source const* source: m_sourceOrder)
 			for (ASTPointer<ASTNode> const& node: source->ast->nodes())
+			{
+				if (!resolver.resolveNamesAndTypes(*node)) return false;
```

Comment:
Newline? ;)

Reply:
In my defense: it didn't have one before ;-).